modules/test/playwright/tests/layout-content-page-editor-web/pages/PageEditorPage.ts

```diff
@@ -33,6 +34,7 @@ export class PageEditorPage {
 		);
 		this.publishButton = page.getByLabel('Publish', {exact: true});
 		this.redoButton = page.getByTitle('Redo');
+		this.toggleSidebarsButton = page.getByLabel('Toggle SidebarsCtrl+⇧+.');
```

Comment:
Yes, it's a bit strange, but the value passed to the button data-title is constructed in this const, which results in that:
```
const getOpenMenuTooltip = (keyLabel) => (
	<>
		<span className="d-block">
			{Liferay.Language.get('toggle-sidebars')}
		</span>

		<kbd className="c-kbd c-kbd-dark mt-1">
			<kbd className="c-kbd">{keyLabel}</kbd>

			<span className="c-kbd-separator">+</span>

			<kbd className="c-kbd">⇧</kbd>

			<span className="c-kbd-separator">+</span>

			<kbd className="c-kbd">.</kbd>
		</kbd>
	</>
);
```

Reply:
sure i will take a look! i think this `getOpenMenuTooltip` is constructed that way because of accessibility patterns maybe? still is a bit strange, i can try to look up more about this to learn more